test/framework/package.py

```diff
@@ -44,16 +44,70 @@
 
 
 MOCKED_FPM = """#!/bin/bash
+
+DEBUG=  #put something here if you want to debug 
+debug_echo () {
+    if [ -n "$DEBUG" ]; then
+        echo "$@" >> /tmp/debug_fpm_mock
```

Comment:
don't hardcode `/tmp/debug_fpm_mock`, use `$TMPDIR` which will be a unique directory when the script it run (you can echo where you're debugging to), or at the very least use `/tmp/$USER/debug_fpm_mock` (but I strongly prefer using `$TMPDIR`)